server/src/schemas/task.schema.ts

```diff
@@ -0,0 +1,47 @@
+import { Types } from 'mongoose';
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+
+import { TaskPriority } from './enums';
+import { timesTampOptions } from './schema.option';
+
+@Schema()
+export class Attachments {
+  @Prop({ required: true })
+  name: string;
+  @Prop({ required: true })
+  type: string;
+  @Prop({ required: true })
+  keyFromBucket: string;
+}
+
+@Schema(timesTampOptions)
+export class Comments {
+  @Prop({ required: true })
+  text: string;
+  @Prop({ type: Types.ObjectId, ref: 'User', required: true })
+  author: Types.ObjectId;
+}
+
+@Schema(timesTampOptions)
+export class Task {
```

Comment:
I would kindly ask you to reorder fields in that way: strings, enums, dates, arrays, foreign column ID